src/expected_loglik.jl

```diff
@@ -73,9 +73,20 @@ function expected_loglik(
     # (see e.g. en.wikipedia.org/wiki/Gauss%E2%80%93Hermite_quadrature)
     xs, ws = gausshermite(gh.n_points)
     # size(fs): (length(y), n_points)
-    fs = √2 * std.(q_f) .* xs' .+ mean.(q_f)
```

Comment:
the `xs'` turns this into a matrix... will this still happen in your broadcasted code? if so, could you add a comment to show where/how it happens?

Reply:
is it because of the _nested_ broadcasting?

Reply:
It is, we are simply doing the broadcasting by hand and avoid allocating every time